modin/pandas/test/dataframe/test_window.py

```diff
@@ -206,6 +206,13 @@ def test_fillna_downcast():
     df_equals(modin_df, result)
 
 
+def test_fillna_4660():
```

Comment:
Why is this added in `test_window.py`?

Reply:
Oh, I found the thread with conversation related this https://github.com/modin-project/modin/pull/4674#discussion_r925605315. So, I resolve this conversation